[all hosts] (sideload) Clarify must select a configure-trust option (#4188)
OfficeDev/office-js-docs-pr: docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md

## Specify the shared folder as a trusted catalog

> There are a couple of options for how you can specify this trust. Follow the instructions for the option that works better for your setup.

Reviewer: ```suggestion
There are two options for how you specify this trust. Follow the instructions for the option that works better for your setup.
```

> - [Configure the trust manually](#configure-the-trust-manually)
> - [Configure the trust with a Registry script](#configure-the-trust-with-a-registry-script)

Reviewer: ```suggestion
- [Configure the trust manually](#configure-the-trust-manually).
- [Configure the trust with a Registry script](#configure-the-trust-with-a-registry-script).
```
